examples/tree_2d_dgsem/elixir_kpp.jl

```diff
@@ -0,0 +1,152 @@
+using Trixi
+using OrdinaryDiffEq
+using Plots
+using LinearAlgebra
+
+###############################################################################
+# Definition of the 2D scalar "KPP equation"
+#
+# See: Kurganov, A., Petrova, G., and Popov, B. (2007).
+# Adaptive Semidiscrete Central-Upwind Schemes for Nonconvex Hyperbolic Conservation Laws,
+# SIAM Journal on Scientific Computing, 29(6), 2381--2401
+
+struct KPPEquation2D <: Trixi.AbstractEquations{2, 1} end
+
+# The KPP flux is F(u) = (sin(u), cos(u))
+@inline function Trixi.flux(u, orientation::Integer, ::KPPEquation2D)
+    if orientation == 1
+        return SVector(sin(u[1]))
+    else
+        return SVector(cos(u[1]))
+    end
+end
+
+# Since the KPP problem is a scalar equation, the entropy-conservative flux is uniquely determined
+@inline function Trixi.flux_ec(u_ll, u_rr, orientation::Integer, ::KPPEquation2D)
+    if abs(u_ll[1] - u_rr[1]) < 1e-12
```

Comment:
Out of curiosity: Did you perform an error analysis of this approximation? Alternatively, did you test using a polynomial approximation whenever both states are close?

Reply:
No, this tolerance is arbitrary. I think you could probably get away with a check for exact equality here. What do you think?

Reply:
If it works for you, just leave it as it is

Reply:
Maybe add a comment here explaining as much, e.g.,
```suggestion
    # The tolerance of 1e-12 is based on experience and somewhat arbitrarily chosen
    if abs(u_ll[1] - u_rr[1]) < 1e-12
```
?